configure.py

```diff
@@ -57,6 +57,11 @@
 with open ('tools/icu/icu_versions.json') as f:
   icu_versions = json.load(f)
 
+sharable_builtins = {'cjs_module_lexer/lexer': 'deps/cjs-module-lexer/lexer.js',
```

Comment:
should it be spelled `shareable` ?

Reply:
ack, will fix that.

Reply:
@targos pushed a commit to fix, thanks for cathing that.